webapp/channels/src/components/admin_console/compliance_reports/compliance_reports.tsx

```diff
@@ -345,7 +345,7 @@ export default class ComplianceReports extends React.PureComponent<Props, State>
                                 defaultMessage='Job Name:'
                             />
                         </label>
-                        <LocalizedInput
+                        <input
                             type='text'
                             className='form-control'
                             id='desc'
```

Comment:
we also have to remove t function usage from the input placeholder. Please refer to the above linked PR to see how to work on this one